src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -80,6 +80,14 @@ module sonic-portchannel {
                         range 1..1024;
                     }
                 }
+               
+               leaf mode {
+                   description "PortChannel Modes values will be from "routed|access|trunk. Default val is routed.";
```

Comment:
Is this the same as port mode? Maybe we can define a type in sonic-types.yang?

Reply:
yes, same as port mode just this is for the PortChannel. I will look into this.

Reply:
modified as per suggestions